### PR TITLE
feat(generativeai): update controlled generation

### PR DESCRIPTION
**Pull request by EmanuelB25**
update controlled generation
https://b.corp.google.com/issues/355233528

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: generative_ai/controlled_generation/response_schema.py [hunk 1]
```diff
@@ -146,11 +146,25 @@ def generate_content3(project_id: str) -> str:
...
+    #                 {"Day": "Wednesday", "Forecast": "Thunderstorms", "Temperature": 68},
+    #                 {"Day": "Thursday", "Forecast": "Cloudy", "Temperature": 66, "Humidity": "60%"},
+    #                 {"Day": "Friday", "Forecast": "Partly Cloudy", "Temperature": 73, "Wind Speed": 12},
+    #                 {"Day": "Saturday", "Forecast": "Sunny", "Temperature": 80, "Humidity": "40%", "Wind Speed": 8}]}
     # [END generativeaionvertexai_gemini_controlled_generation_response_schema_3]
+    return response
 
-    return response.text
+
+
```
Comment (msampathkumar) [inline generative_ai/controlled_generation/response_schema.py:162]: lets remove these 4 lines below :)

File: generative_ai/controlled_generation/response_schema.py [hunk 2]
```diff
@@ -146,11 +146,25 @@ def generate_content3(project_id: str) -> str:
             response_mime_type="application/json", response_schema=response_schema
         ),
     )
-
     print(response.text)
+    # Example reponse:
```
Comment (msampathkumar) [inline generative_ai/controlled_generation/response_schema.py:150]: Please note that the `prompt` above is also changed!

File: generative_ai/controlled_generation/response_schema.py (hunk 1 above)
Comment (EmanuelB25) [reply generative_ai/controlled_generation/response_schema.py]: updated.

File: generative_ai/controlled_generation/response_schema.py (hunk 2 above)
Comment (EmanuelB25) [reply generative_ai/controlled_generation/response_schema.py:152]: updated.

**Review by msampathkumar — APPROVED** [review]
LGTM

**Pull request merged**